repo/plugin.video.nzbdav/resources/lib/resolver_fallback.py

```
@@ -0,0 +1,707 @@
+    return existing_jobs
+
+
+def _submit_one_fallback_candidate(
+    nzb_url, job_name, monitor, settings_getter=None, dead=None
+):
+    """Submit a single fallback candidate; return its ``nzo_id`` or ``None``."""
+    try:
+        nzo_id, submit_error = _resolver.submit_nzb(
+            nzb_url, job_name, **_resolver._settings_getter_kwargs(settings_getter)
+        )
+    except Exception as error:  # pylint: disable=broad-except
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)